clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/description/KubernetesPatchOptions.java

```diff
@@ -29,6 +31,14 @@ public static KubernetesPatchOptions json() {
     return new KubernetesPatchOptions().setMergeStrategy(json);
   }
 
+  public static KubernetesPatchOptions merge() {
```

Comment:
added the other strategies here for convenience. so we can eventually add the strategies into integration tests